taichi/codegen/codegen_llvm.cpp

```diff
@@ -1410,6 +1437,8 @@ void CodeGenLLVM::store_floats_with_shared_exponents(BitStructStoreStmt *stmt) {
     llvm::Value *max_exp_bits = nullptr;
     for (auto f : floats) {
       // TODO: we only support f32 here.
+      //  Shall we set the return type of extract_exponent_from_float to
+      //  bit_struct_physical_type?
```

Comment:
```suggestion
```

Good question - I think the answer is no, since even for f64, which we don't support yet, only has 11-bit exponents :-)